pypesto/objective/amici/amici_util.py

```diff
@@ -194,6 +194,8 @@ def par_index_slices(
             )
             for par_sim_id, par_opt_id in condition_map_sim_var.items()
             if isinstance(par_opt_id, str)
+            # FIXME is it safe to ignore missing ones?
```

Comment:
should be fine, no?

Reply:
I think so. Wasn't sure if it's better / how to avoid, as it could cover up other issues.